whipper/command/cd.py

```diff
@@ -455,7 +461,16 @@ def _ripIfNotRipped(number):
                                     self.ittoc.getTrackLength(number), number)
 
             self.program.saveRipResult()
-
+        try:
+            excluded_tracks = self.options.excluded_tracks.split(",")
+        except UnboundLocalError:
+            excluded_tracks = []
+        excluded_tracks_integer = []
+        for i in excluded_tracks:
```

Comment:
I think it would be better to parse the track listing in the `handle_arguments()` method. I also bet there's a way to tell argparse to expect a variable number of integers, which will save us from having to manually typecast strings to integers.